WaterwitchGUI/src/main.cpp

```diff
@@ -0,0 +1,256 @@
+#include <GLFW/glfw3.h>
+#include "imgui.h"
+#include "backends/imgui_impl_glfw.h"
+#include "backends/imgui_impl_opengl3.h"
+#include <iostream>
+#include "Config.hpp"
+#include "Power.hpp"
+#include <fstream>
+#include <nlohmann/json.hpp>
+#include <filesystem>
+
+using json = nlohmann::json;
+using recursive_directory_iterator = std::filesystem::recursive_directory_iterator;
+
+Config config;
+Power power;
+
+bool showConfigWindow = false;
+bool showCameraWindow = false;
+bool showPilotWindow = false;
+
+int main() {
+    filesystem::create_directory("configs");
```

Comment:
See my comment discussing the place of `profiles_manager.py`. 

We should shift our focus to interfacing with `profiles_manager.py` rather than start from scratch here. Standard convention is to not rewrite existing functionality in the same project (i.e `Software_2025`).